docs/source/releases/index.rst

```diff
@@ -43,6 +34,7 @@ Version 1.12
 
    v1_12_4
    v1_12_3
+   v1_12_0a0
```

Comment:
Why did we add a release note for a release that appears to have no updates?

Reply:
I'll remove that.